ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: s-d-tpu
+spec:
+  serveConfigV2: |
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
+          pip:
+            - pydantic<2
+            - google-api-python-client
+            - pillow
+            - diffusers==0.7.2
+            - transformers==4.24.0
+            - flax
+            - ml_dtypes==0.2.0
+            - jax[tpu]==0.4.11
+            - -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
I haven't seen `-f` before. I am not sure whether this works or not.

Reply:
It just specifies the link to search, from the runtime_env_setup.log I can see that it successfully runts it.